content/articles/spatial_mgrs/index.Rmd

```diff
@@ -0,0 +1,427 @@
+urlcolor: blue
+linkcolor: blue
+output: 
+  md_document:
+    preserve_yaml: true
+    variant: gfm+footnotes
+---
+
+```{r setup, include=FALSE}
+library(sf)
+library(stars)
+library(ggplot2)
+library(dplyr)
+library(rnaturalearth)
+knitr::opts_chunk$set(echo = FALSE, fig.align="center")
+theme_set(theme_bw())
+theme_update(
+  panel.background = element_blank(),
+  panel.ontop = TRUE,
+  panel.grid = element_line(colour = "grey40"),
+  axis.title = element_blank()
+)
+countries <- ne_countries(scale = "large", continent = "Europe", returnclass = "sf")
+countries <- st_transform(countries, "EPSG:3035")
+world <- ne_countries(returnclass = "sf")
+world <- st_transform(world, "ESRI:54032")
+```
+
+```{r eval=FALSE, include=FALSE}
+provinces <- httr::parse_url("https://geo.api.vlaanderen.be/VRBG/wfs") |>
+  purrr::list_merge(query = list(request = "GetFeature",
+                                 typeName = "VRBG:Refprv",
+                                 srsName = "EPSG:3812")) |>
+  httr::build_url() |>
+  read_sf(crs = 3812)
+st_write(provinces, "provinces.gpkg")
+```
+
+```{r include=FALSE}
+provinces <- read_sf("provinces.gpkg")
+```
+
+
+#### Summary
+
+Grids are widely used in biodiversity monitoring to define the basic spatial units for data collection, data processing or mapping.
+Authorative definitions of widely used grids and grid reference systems are not globally coordinated though, contrary to the case of the EPSG-dataset for coordinate reference systems.
+Here I explain differences between the UTM and MGRS grids, provide references to their full definition and refer to some software implementations for MGRS.
+The full definitions are not given here since the primary aim is to draw attention to important properties and differences.
+For the same reason, the UPS portion of MGRS is not further explained here.
+
+This post results from a limited literature study on these topics (see [Bibliography](#bibliography)).
+
+In short:
+
+- The UTM grid and the MGRS grid outside the polar regions are both derived from the UTM (Universal Transverse Mercator) map projection system.
+Both grids have a world-wide coverage, with exception of the polar regions in the case of the UTM grid.
+In the polar regions, the MGRS derives its grid from the UPS (Universal Polar Stereographic) map projection.
+- UTM grid references consist solely of digits (Arabic numerals), while MGRS grid references are alphanumeric (combination of digits & Latin characters).
+- When decreasing the precision of a position, coordinates are to be rounded while in (UTM and MGRS) grid references the numeric parts representing easting and northing are to be truncated.
+- Projected coordinate reference systems (CRSs) specify a geodetic datum so that coordinates can be linked to a physical location on the Earth surface. Likewise, you need to specify (and be aware of) the geodetic datum when using a (UTM or MGRS) grid reference for positioning.
```

Comment:
```suggestion
- Projected coordinate reference systems (CRSs) specify a geodetic datum so that coordinates can be linked to a physical location on the Earth's surface. Likewise, you need to specify (and be aware of) the geodetic datum when using a (UTM or MGRS) grid reference for positioning.
```